### PR TITLE
Add MU/Milestone information to /etc/susemanager-release

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Reimplementatition of https://github.com/uyuni-project/uyuni/pull/7036, addresses @mcalmer's [comment](https://github.com/uyuni-project/uyuni/pull/7036#pullrequestreview-1433812875) 

Eventually the file should maybe be moved into a separate package, that can then be included in all the containers.

### Before

At SUSE Manager:
```
suma-refhead-srv:~ # cat /etc/uyuni-release 
Uyuni release 2023.09
suma-refhead-srv:~ # cat /etc/susemanager-release 
SUSE Manager release 4.4 (2023.09)
```
At Uyuni:
```
uyuni-refmaster-srv:~ # cat /etc/uyuni-release
Uyuni release 2023.09
uyuni-refmaster-srv:~ # cat /etc/susemanager-release
cat: /etc/susemanager-release: No such file or directory
```
### After
At SUSE Manager:
```
suma-refhead-srv:~ # cat /etc/uyuni-release 
cat: /etc/uyuni-release: No such file or directory
suma-refhead-srv:~ # cat /etc/susemanager-release 
SUSE Manager release 4.4 (4.4.0 Alpha1)
```
At Uyuni:
```
uyuni-refmaster-srv:~ # cat /etc/uyuni-release
Uyuni release 2023.09
uyuni-refmaster-srv:~ # cat /etc/susemanager-release
cat: /etc/susemanager-release: No such file or directory
```

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: Not part of the documentation
- [x] **DONE**

## Test coverage
- No tests: No tests

- [x] **DONE**

## Links

Reimplements: https://github.com/uyuni-project/uyuni/pull/7036

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [x] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: web/spacewalk-web.spec [hunk 1]
```diff
@@ -200,6 +201,15 @@ install -m 644 conf/rhn_web.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defa
 install -m 644 conf/rhn_dobby.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults
 install -m 755 modules/dobby/scripts/check-database-space-usage.sh $RPM_BUILD_ROOT/%{_sysconfdir}/cron.daily/check-database-space-usage.sh
 
+if grep -F 'product_name' %{_datadir}/rhn/config-defaults/rhn.conf | grep 'SUSE Manager' >/dev/null; then
+  SUMA_REL=$(echo %{version} | awk -F. '{print $1"."$2}')
+  SUMA_FULL_REL=$(sed -n 's/web\.version\s*=\s*\(.*\)/\1/p' $RPM_BUILD_ROOT%{_datadir}/rhn/config-defaults/rhn_web.conf)
```
Comment (juliogonzalez) [inline web/spacewalk-web.spec:206]: I also improved the version detection a bit. Previous PR was overcomplicating things.

File: web/spacewalk-web.spec [hunk 2]
```diff
@@ -48,8 +48,9 @@ BuildArch:      noarch
 Requires(pre):  uyuni-base-common
 BuildRequires:  gettext
 BuildRequires:  make
-BuildRequires:  uyuni-base-common
 BuildRequires:  perl(ExtUtils::MakeMaker)
+BuildRequires:  spacewalk-backend
+BuildRequires:  uyuni-base-common
```
Comment (juliogonzalez) [inline web/spacewalk-web.spec:53]: This is just a reorder, to make it alphabetical, as requested by the guidelines.

**Comment by juliogonzalez** [comment]
I removed Welder. His review is still welcomed, of course, but not strictly required for this PR.

**Review by mcalmer — APPROVED** [review]

**Comment by juliogonzalez** [comment]
The failed Java tests seem to be unrelated, as they are failing on the branch as well.

**Review by srbarrios — COMMENTED** [review]
Nice! Thank you very much for this improvement!

**Review by deneb-alpha — APPROVED** [review]
LGTM, thanks

**Comment by deneb-alpha** [comment]
I readded  a review  for  frontend because  now it's  required

**Review by parlt91 — APPROVED** [review]

**Review by cbbayburt — APPROVED** [review]

**Pull request merged**